bin/generate.js

```diff
@@ -60,9 +58,9 @@ const parseArgs = () => {
   }
 
   const test = args['--test'];
-  const name = args['--name'] || 'component';
+  const name = args['--name'].split('-')[1];
   const npm = args['--npm'] || '@aurodesignsystem';
-  const namespace = args['--namespace'] || 'auro';
+  const namespace = args['--name'].split('-')[0];
```

Comment:
This causes the script to inelegantly blow up if a non-hyphenated name is used. Perhaps either check for a hyphen and exit if one doesn't exist with a friendly message or auto-add "auro-" if they don't have one.

Reply:
Yup... see updated code in PR.

Reply:
Just noticed in my testing that we should make sure there are no spaces as well. The following command is valid and will generate a component.

```
node bin/generate.js -t -n "auro auro-name name" -P @mypackage -d ./auro-test
```

Reply:
See [commit](https://github.com/AlaskaAirlines/WC-Generator/pull/116/commits/8be2034f84a3ee3db7c2cbaeddeff02e4dea980a)